hoot-core/src/main/cpp/hoot/core/io/OsmGeoJsonReader.cpp

```diff
@@ -63,7 +63,7 @@ using namespace geos::geom;
 namespace hoot
 {
 
-HOOT_FACTORY_REGISTER(OsmJsonReader, OsmGeoJsonReader)
+HOOT_FACTORY_REGISTER(OsmMapReader, OsmGeoJsonReader)
```

Comment:
`OsmMapReader` is the base class of `OsmJsonReader` and is the base factory name.  Without this `OsmGeoJsonReader` isn't tested for support in the factory methods.